### ISSUE TITLE
Domains not working

### ISSUE THREAD

**Issue by kephas** [Scalingo/scalingo-ruby-api#5]
I tried creating a domain but there seems to be an issue in how data is fed to the Faraday:

```
> client.apps.find('my-app').domains.create("example.com")
NoMethodError: undefined method `bytesize' for {:domain=>{:name=>"example.com"}}:Hash
from /usr/local/lib/ruby/2.5.0/net/http/generic_request.rb:183:in `send_request_with_body'
```

**Comment by Soulou** [comment]
Thanks for the issue, patch on its way

**Comment by Soulou** [comment]
Fixed, released in v2.0.1
